indexer/feature.hpp

```diff
@@ -361,10 +369,15 @@ class FeatureType : public FeatureBase
   mutable points_t m_points, m_triangles;
   mutable feature::Metadata m_metadata;
 
+  // @TODO |m_altitudes| should be exchanged with vector<TAltitude>.
+  // If the vector is empty no altitude information is available for this feature.
+  mutable feature::Altitudes m_altitudes;
```

Comment:
This leads to a lot of misunderstanding about how FeatureType works. Please, don't use mutable, really :)